clusters/hosted-mgmt/hive/pools/konflux/konflux-ocp-4-15-0-amd64-aws-us-east-2_clusterpool.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  creationTimestamp: null
+  labels:
+    architecture: amd64
+    cloud: aws
+    owner: konflux
+    product: ocp
+    region: us-east-2
```

Comment:
Is not us-east-1 more cheaper?

Reply:
https://www.concurrencylabs.com/blog/choose-your-aws-region-wisely/
oregon (us-west-2), n. virginia (us-east-1), and ohio (us-east-2) are roughly the same regarding cost

we can use us-east-1 for hcp clusters